app/views/projects/show.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1><%= "#{@project.name}" %></h1>
+
+<p><%= "Material: #{@project.material}" %></p>
+<p><%= "Challenge Theme: #{@challenge.theme}" %></p>
+<p><%= "Number of Contestants: #{@project.contestants.count}" %></p>
```

Comment:
so for these two, you could just say `@project.challenge.theme` and `@project.number_of_contestants` (assuming you make that method)... Although count is super quick to just throw on contestants, it's still some form of calculation, and I'd pull that out into a model method, just like you did for average the contestants average years of experience.